addon/TelemetrySender.js

```diff
@@ -63,6 +63,8 @@ TelemetrySender.prototype = {
   },
 
   _sendPing(data) {
+    console.log(`PAYLOAD: ${data}`);
```

Comment:
I'm not sure, I think we should remove it because it'll log to the browser console when we deploy this and i don't know if that's ok